### ISSUE TITLE
Remover links quebrados

### ISSUE THREAD

**Issue by viguardieiro** [CEPLN/cepln.github.io#1]
Como as páginas foram copiadas diretamente do site antigo, alguns links estão quebrados. É necessários removê-los.

**Comment by arademaker** [comment]
Sim, pode remover os links que vc não conseguir encontrar o novo endereço.

**Comment by viguardieiro** [comment]
Na página Fóruns, praticamente todos os links estão quebrados, exceto pelo da ACL. Como o intuito da página são os links, deixo sem eles ou removo a página?

**Comment by arademaker** [comment]
mantenha estes links quebrados, eu reviso estes casos. Esta página é mais sensível.

**Comment by viguardieiro** [comment]
Removi os seguintes links:
- Home:
	http://www.sbc.de9.ime.eb.br/
- Mapeamento da comunidade:
	http://www.d.umn.edu/~tpederse/registry/registry.cgi
	http://www.nilc.icmc.usp.br/cepln/STIL2009-Painel-PardoEtAl.pdf/
- Periódicos:
	http://www.eup.ed.ac.uk/journals/content.aspx?pageId=1&journalId=12801
	http://www.asis.org/Publications/JASIS/jasis.html
	http://web.stanford.edu/group/cslipublications/cslipublicationsLiLT
	http://www.dcs.kcl.ac.uk/journals/rolc/
	https://www.computer.org/portal/site/transactions/index.jsp?pageID=tkde_home
	https://www.computer.org/portal/site/transactions/index.jsp?pageID=tpami_home
	http://www.mcs.anl.gov/JAR/
	http://springerlink.metapress.com/content/1573-0565/

Ainda estão:
- Fóruns:
	https://grupos.ufrgs.br/mailman/listinfo/ce-pln
	https://grupos.ufrgs.br/mailman/listinfo/Ceia-l/
	https://grupos.ufrgs.br/mailman/listinfo/Sbc-l/
	https://lists.fct.unl.pt/mailman/listinfo/forum-lp
	http://tech.groups.yahoo.com/group/corpus-brasil/

**Comment by viguardieiro** [comment]
Na página Fóruns, ainda estão quebrados:

- http://grupos.ufrgs.br/mailman/listinfo/Sbc-l/
- https://lists.fct.unl.pt/mailman/listinfo/forum-lp

Posso removê-los?
